h2o-algos/src/test/java/hex/ensemble/StackedEnsembleTest.java

```diff
@@ -1098,5 +1099,66 @@ public void testInvalidFoldColumn_trainingFrame() {
     }
   }
 
+  @Test
+  public void testBaseModelsWorkWithGrid() {
+    GBMModel gbmModel = null;
+    try {
+      Scope.enter();
+
+      final Frame trainingFrame = TestUtil.parse_test_file("./smalldata/junit/weather.csv");
+      Scope.track(trainingFrame);
+      trainingFrame.toCategoricalCol("RainTomorrow");
+
+      GBMModel.GBMParameters parameters = new GBMModel.GBMParameters();
+      parameters._train = trainingFrame._key;
+      parameters._seed = 0xFEED;
+      parameters._response_column = "RainTomorrow";
+      parameters._ntrees = 1;
+      parameters._keep_cross_validation_predictions = true;
+
+      GBM gbm = new GBM(parameters);
+      gbmModel = gbm.trainModel().get();
+      assertNotNull(gbmModel);
+
+      final Integer[] ntreesArr = new Integer[]{5, 50, 7, 8, 9, 10};
+      final Integer[] maxDepthArr = new Integer[]{2, 3, 4};
```

Comment:
please keep it minimal for tests: 3-4 models in the grid is enough here, not need for 18.